### ISSUE TITLE
LOOT giving strange error

### ISSUE THREAD

**Issue by DigitalWatcher** [loot/loot#1866]
I don't know why I keep getting this error from LOOT when I try to update the master list or sort the plugins.
![RedditProof](https://github.com/loot/loot/assets/134321957/9c211b25-bcdc-468a-b867-1d0241618c2e)

**Comment by pStyl3** [comment]
@Ortham The error message should ideally be more specific than `is not a regular file`.

@DigitalWatcher Please go to LOOTs settings, enable debug logging, apply the change & try to update the masterlist again. Please provide us with the resulting `LOOTDebugLog.txt` file that can be found in: `%localappdata%\LOOT`

**Comment by Ortham** [comment]
> @Ortham The error message should ideally be more specific than `is not a regular file`.

It's not intentional, I guess something is missing the handling that turns errors into the generic "something went wrong" message that gets displayed.

**Comment by DigitalWatcher** [comment]
> @Ortham The error message should ideally be more specific than `is not a regular file`.
> 
> @DigitalWatcher Please go to LOOTs settings, enable debug logging, apply the change & try to update the masterlist again. Please provide us with the resulting `LOOTDebugLog.txt` file that can be found in: `%localappdata%\LOOT`

Heres the debug file.

[LOOTDebugLog.txt](https://github.com/loot/loot/files/11539641/LOOTDebugLog.txt)

**Comment by sibir-ine** [comment]
> Heres the debug file.
> 
> [LOOTDebugLog.txt](https://github.com/loot/loot/files/11539641/LOOTDebugLog.txt)

Just to make sure, is this the resulting debug log after receiving the error again? Also, is it the full, unedited log? It looks like it got cut off at the end:

```
[10:10:45.172223] [info]: Updating file at %USERPROFILE%\AppData\Local\LOOT\games\Skyrim Special Edition\masterlist.yaml using file at 
```

**Comment by DigitalWatcher** [comment]
> > Heres the debug file.
> > [LOOTDebugLog.txt](https://github.com/loot/loot/files/11539641/LOOTDebugLog.txt)
> 
> Just to make sure, is this the resulting debug log after receiving the error again? Also, is it the full, unedited log? It looks like it got cut off at the end:
> 
> ```
> [10:10:45.172223] [info]: Updating file at %USERPROFILE%\AppData\Local\LOOT\games\Skyrim Special Edition\masterlist.yaml using file at 
> ```

Yes I directly pasted it

**Comment by pStyl3** [comment]
The log includes this line:
> [10:10:38.127011] [error]: Error while checking if game "Fallout 3" is installed through the Epic Games Store: directory_iterator::directory_iterator: The system cannot find the path specified.: "C:\ProgramData\Epic\EpicGamesLauncher\Data\Manifests"

@DigitalWatcher Do you have Fallout 3 installed right now, or at some previous point, through the Epic Games Store or through a different distributor? If yes, can you tell in which exact folder `Fallout3.exe` was?

**Comment by DigitalWatcher** [comment]
> The log includes this line:
> 
> > [10:10:38.127011] [error]: Error while checking if game "Fallout 3" is installed through the Epic Games Store: directory_iterator::directory_iterator: The system cannot find the path specified.: "C:\ProgramData\Epic\EpicGamesLauncher\Data\Manifests"
> 
> @DigitalWatcher Do you have Fallout 3 installed right now, or at some previous point, through the Epic Games Store or through a different distributor? If yes, can you tell in which exact folder `Fallout3.exe` was?

I did on a different computer but not on this one

**Comment by Ortham** [comment]
I think what's happening is that the masterlist source setting is empty so it's not a valid URL, so it's assumed to be a file path.

@DigitalWatcher can you share your `%LOCALAPPDATA%\LOOT\settings.toml` file?

**Comment by DigitalWatcher** [comment]
Here's the file 
[settings.zip](https://github.com/loot/loot/files/11546859/settings.zip)

**Comment by DigitalWatcher** [comment]
> I think what's happening is that the masterlist source setting is empty so it's not a valid URL, so it's assumed to be a file path.
> 
> @DigitalWatcher can you share your `%LOCALAPPDATA%\LOOT\settings.toml` file?

It is empty the masterlist source path, What path should i set for it?

**Comment by DigitalWatcher** [comment]
I guess the master list is empty? what link should i add there ?
![masterlistempty](https://github.com/loot/loot/assets/134321957/784057a3-e5fe-45d6-b346-318f57defd0a)

**Comment by sibir-ine** [comment]
`https://raw.githubusercontent.com/loot/skyrimse/v0.18/masterlist.yaml`

**Comment by Ortham** [comment]
Fixed as of cb83a26aac80b61c63e9747ed2477cf90c1d5863.
